tests/e2e/hcp_machine_pool_test.go

```diff
@@ -202,32 +201,43 @@ var _ = Describe("HCP MachinePool", ci.Day2, ci.NonClassicCluster, ci.FeatureMac
 	It("can be created with security groups - [id:73068]", ci.High,
 		func() {
 			By("Prepare additional security groups")
-			sgService := exe.NewSecurityGroupService()
+			sgService := exec.NewSecurityGroupService()
 			output, err := sgService.Output()
 			Expect(err).ToNot(HaveOccurred())
 			if output.SGIDs == nil {
 
-				sgArgs := &exe.SecurityGroupArgs{
+				sgArgs := &exec.SecurityGroupArgs{
 					AWSRegion: profile.Region,
 					VPCID:     vpcOutput.VPCID,
 					SGNumber:  4,
 				}
 				err = sgService.Apply(sgArgs, true)
 				Expect(err).ToNot(HaveOccurred())
-				// defer sgService.Destroy()
+				defer sgService.Destroy()
 			}
 
 			output, err = sgService.Output()
 			Expect(err).ToNot(HaveOccurred())
 
 			replicas := 0
 			machineType := "r5.xlarge"
-			name := "ocp-73068-2"
+			name := "ocp-73068"
 			sgIDs := output.SGIDs
 			if len(sgIDs) >= 4 {
 				sgIDs = sgIDs[0:4]
 			}
-			MachinePoolArgs := &exe.MachinePoolArgs{
+
+			// workaround
+			By("Check cluster tags and set it to the machinepool")
+			resp, err := cms.ListNodePools(ci.RHCSConnection, clusterID)
+			Expect(err).ToNot(HaveOccurred())
+			npDetail, err := cms.RetrieveNodePool(ci.RHCSConnection, clusterID, resp[0].ID(), map[string]interface{}{
+				"fetchUserTagsOnly": true,
+			})
+			Expect(err).ToNot(HaveOccurred())
+			workAroundTags := npDetail.AWSNodePool().Tags()
```

Comment:
I guess at some point we should have a `helper.RetrieveClusterUserTags` method. wdyt ?

Reply:
@radtriste Make sense. But right now it's not valuable for a workaround